opencti-platform/opencti-front/src/private/components/workspaces/WorkspaceHeader.jsx

```diff
@@ -139,6 +149,19 @@ const WorkspaceHeader = ({
   const handleOpenManageAccess = () => setDisplayManageAccess(true);
   const handleCloseManageAccess = () => setDisplayManageAccess(false);
 
+  const handleDownloadAsStixReport = () => {
+    fetchQuery(workspaceHeaderToStixReportBundleQuery, { id: workspace.id })
+      .toPromise()
+      .then((data) => {
+        const toStixBundleData = data?.workspace?.toStixReportBundle;
+        if (toStixBundleData) {
+          const blob = new Blob([toStixBundleData], { type: 'text/json' });
+          const fileName = `${workspace.name}-export-stix-report`;
```

Comment:
It's maybe better for the name to be close to the actual export file (connector side) :
![image](https://github.com/OpenCTI-Platform/opencti/assets/53513584/d7d3f532-2077-40d3-b4a7-b969502e2a27)